database/error.go

```diff
@@ -3,16 +3,16 @@ package database
 import "github.com/layer5io/meshkit/errors"
 
 var (
-	ErrNoneDatabaseCode           = "11010"
-	ErrDatabaseOpenCode           = "11011"
-	ErrSQLMapUnmarshalJSONCode    = "11012"
-	ErrSQLMapUnmarshalTextCode    = "11013"
-	ErrSQLMapMarshalValueCode     = "11014"
-	ErrSQLMapUnmarshalScannedCode = "11015"
-	ErrSQLMapInvalidScanCode      = "11016"
-
-	ErrNoneDatabase      = errors.New(ErrNoneDatabaseCode, errors.Alert, []string{"No Database selected"}, []string{}, []string{"database name is empty"}, []string{"Input a name for the database"})
-	ErrSQLMapInvalidScan = errors.New(ErrSQLMapUnmarshalScannedCode, errors.Alert, []string{"invalid data type: expected []byte"}, []string{}, []string{}, []string{})
+	ErrNoneDatabaseCode              = "11010"
+	ErrDatabaseOpenCode              = "11011"
+	ErrSQLMapUnmarshalJSONCode       = "11012"
+	ErrSQLMapUnmarshalTextCode       = "11013"
+	ErrSQLMapMarshalValueCode        = "11014"
+	ErrSQLMapUnmarshalScannedCode    = "11015"
+	ErrSQLMapInvalidScanCode         = "11016"
+	ErrClosingDatabaseConnectionCode = "11017"
+	ErrNoneDatabase                  = errors.New(ErrNoneDatabaseCode, errors.Alert, []string{"No Database selected"}, []string{}, []string{"database name is empty"}, []string{"Input a name for the database"})
+	ErrSQLMapInvalidScan             = errors.New(ErrSQLMapUnmarshalScannedCode, errors.Alert, []string{"invalid data type: expected []byte"}, []string{}, []string{}, []string{})
```

Comment:
Please enhance this error message.

Reply:
You mean rename the error code? What is there to enhance exactly here? I didn't quite get you

Reply:
Oh, a probable cause and suggested remediation are missing.

Reply:
My PR only added line 13, which is just a code. And the error that I added does have both of the asked things.

Reply:
Oh, good. Ok. You're directly off the hook here, then.

Indirectly, you're on the hook. I encourage you to think holistically for the health of the project.